tools-ui/src/main/java/com/hedera/hashgraph/client/ui/MainController.java

```diff
@@ -635,19 +635,20 @@ public void displaySystemMessage(final Exception exception) {
 	public String getVersion() {
 		final var buildProps = new Properties();
 		var version = UNKNOWN_VERSION;
-		try {
-			buildProperties(buildProps);
-			final var formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
-			final var dateTime = formatter.parse(buildProps.get("git.build.time").toString());
-			formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
-			final var dateTimeUTC = formatter.format(dateTime);
-			version = String.format("Version: %s, UTC-BUILD-TIME: %s, COMMIT-ID: %s",
-					buildProps.get("git.build.version"),
-					dateTimeUTC, buildProps.get("git.commit.id.abbrev"));
-		} catch (final Exception ex) {
-			LOG.error("Error Printing Version: {}", ex.getMessage());
-			displaySystemMessage(ex);
-		}
+		// build.properties file doesn't exist, maybe part of the maven build
+//		try {
+//			buildProperties(buildProps);
+//			final var formatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+//			final var dateTime = formatter.parse(buildProps.get("git.build.time").toString());
+//			formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
+//			final var dateTimeUTC = formatter.format(dateTime);
+//			version = String.format("Version: %s, UTC-BUILD-TIME: %s, COMMIT-ID: %s",
+//					buildProps.get("git.build.version"),
+//					dateTimeUTC, buildProps.get("git.commit.id.abbrev"));
+//		} catch (final Exception ex) {
+//			LOG.error("Error Printing Version: {}", ex.getMessage());
+//			displaySystemMessage(ex);
+//		}
```

Comment:
It looks like the "Git-Commit-Id-Plugin" generates this file. I did a quick search and it looks like you can use this plugin (https://plugins.gradle.org/plugin/com.gorylenko.gradle-git-properties) with gradle to generate a similar file. I think it would be good to sort this out prior to merging this PR (I'm OK with disabling tests for the moment, but I would prefer not losing functionality in the app in this PR)